src/components/portrait/index.d.ts

```diff
@@ -0,0 +1,6 @@
+export { default } from "./portrait";
+export * from "./portrait";
+export { default as PortraitGravatar } from "./portrait-gravatar";
+export * from "./portrait-gravatar";
+export { default as PortraitInitials } from "./portrait-initials";
+export * from "./portrait-initials";
```

Comment:
`index.d.ts` should reflect what is in the `index.js` file. User will not need other components to be exported here.
```suggestion
export { default } from "./portrait";
```